docs/index.md

```diff
@@ -28,56 +28,23 @@ Since lakeFS is compatible with the S3 API, all popular applications will work w
 
 ![lakeFS s3 addressing](assets/img/s3_branch.png)
 
-Here's why you need it:
+## Here's why you need it:
 
-### Fragile Writers
+### Developer Environment for Data
+* **Experimentation** - try tools, upgrade versions and evaluate code changes in isolation. By creating a branch of the lake you get an isolated snapshot of the lake you can experiment with while others are not exposed. You can compare branches with different experiments or compare your branch to the master lake.  
+* **Reproducibility** - go back to any point of time to a consistent version of your data lake. By making commit history available for a configurable duration - you can read from the lake at any given point in time, compare changes made, and safely rollback if necessary.
 
-Writing to object stores is simple, scalable and cheap but could also be error prone, for example:
+### Continuous Data Integration
+* **Ingest new data safely by enforcing best practices** - make sure new data sources adhere to your lake’s best practices such as format and schema enforcement, naming convention, etc’. By consuming data to an isolated branch and creating pre-merge hooks you can define automated rules and tests that are required to pass before introducing new data sources.
```

Comment:
```suggestion
* **Ingest new data safely by enforcing best practices** - make sure new data sources adhere to your lake’s best practices such as format and schema enforcement, naming convention, etc. By consuming data to an isolated branch and creating pre-merge hooks you can define automated rules and tests that are required to pass before introducing new data sources.
```

"By consuming data to an isolated branch" doesn't sound right. Maybe "By inserting data"?

Reply:
or ingest like below? TBH I'm not sure what ingest really means